packages/react-components/review/stable/react-components.api.md

```diff
@@ -56,7 +56,11 @@ export type AnnouncerProps = {
 };
 
 // @beta
-export type AttachmentMetadata = FileMetadata;
+export interface AttachmentMetadata {
+    extension: string;
+    name: string;
+    url: string;
```

Comment:
how about `id`, `payload` ? are these 2 never GA-ed?

Reply:
> how about `id`, `payload` ? are these 2 never GA-ed?

Yes id and payload is under the feature flag while the rest somehow was released to GA.

Reply:
> how about `id`, `payload` ? are these 2 never GA-ed?

Actually this is the react **components** api view not the react **communication** one.